packages/vue/vue-vanilla/src/array/ArrayListRenderer.vue

```diff
@@ -77,6 +84,12 @@ const controlRenderer = defineComponent({
     noData(): boolean {
       return !this.control.data || this.control.data.length === 0;
     },
+    arraySchema(): JsonSchema | undefined {
+      return Resolve.schema(
+          this.control.rootSchema,
```

Comment:
The first parameter should be the `props.schema` otherwise this will fail for nested objects/arrays.